ReactTask/src/hooks/useActionsWithRedux.ts

```diff
@@ -19,9 +24,21 @@ interface IProfile {
   };
 }
 
+interface IPatientState {
+  patientList: {
+    resolutions: IPatientResolutions[];
+  };
+}
+
 function useActionsWithRedux() {
   const dispatch = useDispatch();
   const profile = useSelector((state: IProfile) => state.authForm.user);
+  const resolutionsPatient = useSelector(
+    (state: IPatientState) => state.patientList.resolutions
+  );
+  const alertMessage = useSelector(
+    (state: IAlertMessage | any) => state.alertMessage
```

Comment:
Why or any?

Reply:
Because I did not fully understand how to correctly declare the interface at this point.
Fixed